mock/docs/site-defaults.cfg

```diff
@@ -164,9 +164,16 @@
 # config_opts['createrepo_on_rpms'] = False
 # config_opts['createrepo_command'] = '/usr/bin/createrepo_c -d -q -x *.src.rpm'
 
-# You can configure which tar is used (for root cache and SCM plugin)
-# valid options are: "gnutar" or "bsdtar"
-# config_opts['tar'] = "gnutar"
+
+# What tar binary should be used by Mock.
+#config_opts['tar_binary'] = "/bin/tar"
```

Comment:
Should the default be /usr/bin/tar ?

Reply:
I'm not sure.  On Fedora, we have the /bin -> /usr/bin link, on Debian there used to be a different approach and I recall that using just /bin is safer?  Try e.g. on `podman run --rm -ti debian:11`.